src/index.js

```diff
@@ -0,0 +1,219 @@
+    })
+    .then((response) => {
+        state.tempValue = convertK2F(response.data.main.temp);
+        document.getElementById("temp-value").innerHTML = state.tempValue;
+        handleTempChanged();
+    })
+
+    .catch((error) => {
+        console.log("There's an error " + error);
+    })
+};
+
+const convertK2F = (temp) => {
+    return Math.trunc((temp - 273.15) * 9 / 5 + 32);
+}
+
+const handleSearchButtonClicked = () => {
+    changeCityHeader();
+    getLocation();
+};
+
+const handleResetBtnClicked = () => {
+    location.reload();
+}
+
+const handleUpBtnClicked = () => {
+    state.tempValue += 1;
+    console.log(state.tempValue);
+
+    document.getElementById("temp-value").innerHTML = state.tempValue;
+    handleTempChanged();
+}
+
+const handleDownBtnClicked = () => {
+    state.tempValue -= 1;
+    console.log(state.tempValue);
+
+    document.getElementById("temp-value").innerHTML = state.tempValue;
+    handleTempChanged();
+}
+
+const handleTempChanged = () => {
+    if (state.tempValue >= 80) {
+        document.getElementById("temp-value").style.color = '#ff0000';
+    } else if (state.tempValue >= 70 && state.tempValue <= 79) {
+        document.getElementById("temp-value").style.color = '#FFA500';
+    } else if (state.tempValue >= 60 && state.tempValue <= 69) {
+        document.getElementById("temp-value").style.color = '#FFFF00';
+    } else if (state.tempValue >= 50 && state.tempValue <= 59) {
+        document.getElementById("temp-value").style.color = '#00FF00';
+    } else if (state.tempValue <= 49) {
+        document.getElementById("temp-value").style.color = '#008080';
+    }
+    console.log(document.getElementById("temp-value").style.color);
+}
+
+const registerEvents = () => {
+    state.citySearchButton.addEventListener("click", handleSearchButtonClicked);
+    state.resetButton.addEventListener("click", handleResetBtnClicked);
+    state.downbutton.addEventListener("click", handleDownBtnClicked);
+    state.upbutton.addEventListener("click", handleUpBtnClicked);
+};
+
+const refreshUI = () => {
+    document.getElementById("temp-value").innerHTML = state.tempValue;
+};
+
+const loadControls = () => {
+    state.citySearchButton = document.getElementById("search-button"); 
+    state.resetButton = document.getElementById("reset-button");
+    state.upbutton = document.getElementById("up-button");
+    state.downbutton = document.getElementById("down-button");
+};
+
+const onLoad = () => {
+    loadControls();
+    registerEvents();
+    refreshUI();
+    console.log(state.tempValue);
+};
+
+onLoad();
+
+// axios
+//     .get('https://us1.locationiq.com/v1/search?key=YOUR_ACCESS_TOKEN&q=SEARCH_STRING&format=json')
+//     .then((response) => {
+
+//     })
+//     .catch((error) => {
+
+//     });
+
+
+// additional weather data:
+// let city = 'london'
+// $.ajax({
+//     method: 'GET',
+//     url: 'https://api.api-ninjas.com/v1/weather?city=' + city,
+//     headers: { 'X-Api-Key': 'YOUR_API_KEY'},
```

Comment:
Remember to remove debugging log statements & comments

Reply:
Removed- two of them were additional features we were working on- the additional weather data and animation.